packages/design-system-tokens/src/themes/medicare-component.ts

```diff
@@ -302,8 +301,8 @@ export const components: AnyTokenValues = {
     '__border-color':                             t.color['border'],
     '-close__color':                              t.color['black'],
     '-header__background-color':                  t.color['transparent'],
-    '-footer__background-color':                  t.color['secondary-lightest'],
-    '-toggle__background-color--hover':           t.color['secondary'],
+    '-footer__background-color':                  t.color['info-lightest'],
```

Comment:
That's fair; I guess it's more nuanced than I thought. I thought they really did intend to use info blue in all themes because the footer was informational. Leaving this as is will force us to address the design intention in the future at some point. I'll revert it.